examples/Stockagent/main.py

```
@@ -0,0 +1,208 @@
+            for sell_action in stock_deals["sell"][:]:
+                if action["price"] == sell_action["price"]:
+                    # 交易成交
+                    close_amount = min(action["amount"], sell_action["amount"])
+                    get_agent(all_agents, action["agent"]).buy_stock(stock.name, close_amount, action["price"])
+                    if not sell_action["agent"] == -1:  # B发行
+                        get_agent(all_agents, sell_action["agent"]).sell_stock(stock.name, close_amount, action["price"])
+                    stock.add_session_deal({"price": action["price"], "amount": close_amount})
+                    create_trade_record(action["date"], session, stock.name, action["agent"], sell_action["agent"],
+                                        close_amount, action["price"])
+
+                    if action["amount"] > close_amount:  # 买单未结束，卖单结束，继续循环
```

Ruff (0.14.3)

Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)